pkg/operator/ceph/object/spec.go

```diff
@@ -360,24 +366,25 @@ func (c *clusterConfig) makeDaemonContainer(rgwConfig *rgwConfig) (v1.Container,
 		return v1.Container{}, errors.Wrap(err, "failed to generate default readiness probe")
 	}
 
+	args := append(
```

Comment:
How about just move lines 369-379 inside the `getRGWContainerCMD()` method? Then all cmd args are in one method.

Reply:
I'd alternatively suggest that we can continue to use args just as they were before. They will get passed to the bash script, and if we use `exec "$@"` as the nginx startup script does, the args will be handled appropriately and won't need escaped.